content/hi/docs/reference/glossary/image.md

```diff
@@ -0,0 +1,17 @@
+---
+title: इमेज (Image)
+id: image
+date: 2018-04-12
+full_link: 
+short_description: >
+  एक कंटेनर का स्टोर्ड इंस्टेंस, जिसमें किसी एप्लिकेशन को चलाने के लिए आवश्यक सॉफ़्टवेयर का सेट हो।
+
+aka: 
+tags:
+- fundamental
+---
+ एक {{< glossary_tooltip text="कंटेनर" term_id="container" >}} का स्टोर्ड इंस्टेंस, जिसमें किसी एप्लिकेशन को चलाने के लिए आवश्यक सॉफ़्टवेयर का सेट हो। 
+
+<!--more-->
+
+सॉफ़्टवेयर को पैकेज करने का एक तरीका, जो इसे कंटेनर रजिस्ट्री में स्टोर्ड करने, स्थानीय सिस्टम पर पुल करने, और एक एप्लिकेशन के रूप में चलाने की अनुमति देता है। इमेज में मेटा डेटा शामिल होता है, जो यह संकेत कर सकता है कि कौन सा एक्ज़ीक्यूटेबल चलाना है, इसे किसने बनाया, और अन्य जानकारी।
```

Comment:
```suggestion
सॉफ़्टवेयर को पैकेज करने का एक तरीका, जो इसे कंटेनर रजिस्ट्री में स्टोर्ड करने, स्थानीय सिस्टम पर पुल (pull) करने, और एक एप्लिकेशन के रूप में चलाने की अनुमति देता है। इमेज में मेटा डेटा शामिल होता है, जो यह संकेत कर सकता है कि कौन सा एक्ज़ीक्यूटेबल चलाना है, इसे किसने बनाया, और अन्य जानकारी प्रदान कर सकता है।
```

Reply:
@niranjandarshann Sure!! Made changes as you suggested!! Thanks!